xbmc/video/dialogs/GUIDialogVideoManagerExtras.cpp

```diff
@@ -156,3 +151,21 @@ int CGUIDialogVideoManagerExtras::NewVideoAssetId(const CFileItem& item)
   return m_database.AddVideoVersionType(typeNewVideoVersion, VideoAssetTypeOwner::AUTO,
                                         VideoAssetType::EXTRA);
 }
+
+CGUIDialogVideoManager::ManagerMessages CGUIDialogVideoManagerExtras::InitializeMessages()
+{
+  ManagerMessages messages;
+  messages.ManagerTitleId = 40025;
+  messages.RemoveTitleId = 40039;
+  messages.RemoveTextId = 40040;
+  messages.AddDialogTitleId = 40015;
+  messages.AddNoDefaultMovieVersionId = 40034;
+
+  return messages;
+}
+
+const CGUIDialogVideoManager::ManagerMessages& CGUIDialogVideoManagerExtras::DialogMessages()
+{
+  static ManagerMessages msg{InitializeMessages()};
```

Comment:
I don't think you need the `InitializeMessages` method.

Doesn't `static ManagerMessages msg{InitializeMessages{40025, 40039, 40040, 40015, 40034}};` work? Maybe you need to define a respective ctor for `ManagerMessages`.

Reply:
I though about doing that, but it's not easy to read.

Reply:
It’s “standard” with modern c++ I’d say and used widely.